src/tempo.py

```diff
@@ -265,7 +242,51 @@ def is_ready(self):
             return False
         return out == "ready"
 
-    def _build_receivers_config(self, receivers: Sequence[ReceiverProtocol]):  # noqa: C901
+    def _build_querier_config(self):
+        """Build querier config"""
+        # TODO this won't work for distributed coordinator where query frontend will be on a different unit
+        return Querier(
+            frontend_worker=FrontendWorker(
```

Comment:
```suggestion
            frontend_worker=FrontendWorkerConfig(
```

or, if you don't like the idea of having 100 objects all called SomethingConfig, I'd be fine with doing:
```
from xx import tempo_config
tempo_config.Querier
tempo_config.Server
```
and so on. Then the prefix makes it clear we're talking about the configuration of a server, and not the server itself.